src/users/users.service.ts

```
@@ -1,78 +1,71 @@
+        }
+        return user;
     }
-    return user;
-  }
 
-  async updateUser(id: string, params: UpdateUserInput) {
-    const rentalIds =
-      params.rentalIds && params.rentalIds.map((id) => new Types.ObjectId(id));
-    delete params['rentalIds'];
+    async updateUser(id: string, params: UpdateUserInput): Promise<UserDocument> {
+        const rentalIds = params.rentalIds && params.rentalIds.map(id => new Types.ObjectId(id));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)